POLYFILLS.md

```diff
@@ -87,36 +142,81 @@ the rest of your application code:
     <title>Polyfill Example</title>
 
     <!-- The following libraries and polyfills are recommended to maximize browser support -->
```

Comment:
```suggestion
    <!-- The following libraries and polyfills are provided to maximize browser support -->
```

Some are required, some are recommended. I also thought about adding a comment "in order to support the last 2 versions of each major browser, as well as IE11".

Reply:
We do not actually provide any of these libraries. They are not dependencies of the `@google/model-viewer` NPM package, so they will not be installed when our module is installed in typical use cases.